### DISCUSSION TITLE
请问大佬（无法设置参数错误）：error: the following arguments are required: --output_dir

### DISCUSSION THREAD

**Discussion by ByteCaprice** [PaddlePaddle/PaddleNLP#5379] [General]
请问这个问题怎么处理，三个类（ModelArguments, DataArguments, CompressionArguments）都没有output_dir这个参数，但用addargument函数加参数以及在三个类的源码里加参数还都添加不进去，显示跟原来的output_dir冲突了，可是代码里根本没有设置这个参数啊，有没有大佬碰到这个问题的

**Comment by Miamas777** [comment]
I also have this problem
